pulsar-broker/src/main/java/org/apache/pulsar/broker/service/AbstractTopic.java

```diff
@@ -1419,4 +1424,40 @@ public static Optional<ClusterUrl> getMigratedClusterUrl(PulsarService pulsar, S
         }
         return Optional.empty();
     }
+
+    private CompletableFuture<Void> catchCreateTopicForRemoteCluster(CompletableFuture<Void> createTopicFuture) {
+        return createTopicFuture.exceptionally(ex -> {
+            Throwable throwable = FutureUtil.unwrapCompletionException(ex);
+            if (throwable instanceof ConflictException) {
+                int code = ((ConflictException) throwable).getStatusCode();
+                if (code == Status.CONFLICT.getStatusCode()) {
```

Comment:
Maybe all the partitioned-topic tries to create topic for the remote cluster, so there will throw `Conflict` exception, right?

Reply:
And also existed in the remote cluster?

Reply:
Since load topic could try to create the topic in the remote cluster, the `create` operation and the error log may confuse the user. So it's better to `query` before `create`.
Another case is: if the user updates the partition in the local cluster(like updating partition from 3~5), the remote can't update.